### PR TITLE
Feature/board acceptance

### PR DESCRIPTION
**Pull request by raulchoque-eng**
implemented feature Board with the post method

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -47,7 +47,10 @@ task cucumber() {
 
 // Dependencies.
 dependencies {
-    testCompile 'io.cucumber:cucumber-java:4.3.1'
+    // https://mvnrepository.com/artifact/io.cucumber/cucumber-java
+    compile group: 'io.cucumber', name: 'cucumber-java', version: '4.7.1'
+
```
Comment (JosueRodriguezGarcia) [inline build.gradle]: delete line 52.

Comment (MadayAlcala) [inline build.gradle]: Please delete the unnecessary line.

File: src/test/java/com/foundation/trello/model/FactoryRequest.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+public final class FactoryRequest {
+
+    public static final String MESSAGE_FOR_UNKNOWN_METHOD = "Unknown method type";
+
+    /**
+     * This method constructor initializes variables.
+     */
+    private FactoryRequest() {
+    }
+    /**
```
Comment (JosueRodriguezGarcia) [inline src/test/java/com/foundation/trello/model/FactoryRequest.java:18]: add new line

File: build.gradle (hunk 1 above)
Comment (raulchoque-eng) [reply build.gradle]: okay thank you, it was resolved

Comment (raulchoque-eng) [reply build.gradle]: this yet was resolved.

File: src/test/java/com/foundation/trello/model/FactoryRequest.java (hunk 2 above)
Comment (raulchoque-eng) [reply src/test/java/com/foundation/trello/model/FactoryRequest.java:18]: okay thank you, it was resolved

File: src/test/java/com/foundation/trello/model/FactoryRequest.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
+package com.foundation.trello.model;
+
+/**
+ * FactoryRequest class.
+ *
+ * @author Raul Choque
+ * @version 0.0.1
+ */
+public final class FactoryRequest {
+
```
Comment (MadayAlcala) [inline src/test/java/com/foundation/trello/model/FactoryRequest.java]: Unnecessary line, please delete.

**Review by JosueRodriguezGarcia — APPROVED** [review]

File: src/test/java/com/foundation/trello/model/FactoryRequest.java (hunk 3 above)
Comment (raulchoque-eng) [reply src/test/java/com/foundation/trello/model/FactoryRequest.java]: Okay thank you it was fixed

**Review by MadayAlcala — APPROVED** [review]
Ready to merge.

**Pull request merged**